c-api-examples/streaming-paraformer-c-api.c

```diff
@@ -0,0 +1,136 @@
+// c-api-examples/streaming-paraformer-c-api.c
```

Comment:
Also, please update
https://github.com/k2-fsa/sherpa-onnx/blob/master/.github/workflows/c-api.yaml
to test the added files.